app/components/app_vaccinate_form_component.rb

```diff
@@ -36,6 +36,16 @@ def campaign_name
   end
 
   def vaccination_initial_delivery_sites
-    %w[left_arm right_arm other]
+    [
+      OpenStruct.new(
+        value: "left_arm_upper_position",
+        label: "Left arm (upper position)"
```

Comment:
Can we easily use existing entries in `en.yml` for this?

```
en:
  activerecord:
    attributes:
      vaccination_record:
        delivery_sites:
          left_arm_upper_position: Left arm (upper position)
```

Reply:
Hmm, not easily but may as well use the translation strings.